lighthouse-core/audits/seo/link-text.js

```diff
@@ -25,6 +25,25 @@ const BLOCKLIST = new Set([
   '続きを読む',
   '続く',
   '全文表示',
+  // Spanish
+  'click aquí',
```

Comment:
There's a fair amount of duplication with the permutations, I wonder if we could do some replacements to keep the signal in here high?


could be as simple as an easy map for now

```js
const CHARACTER_REPLACEMENTS = new Map([
  [/í/g, 'i'],
  [/á/g, 'a'],
  [/ó/g, 'o'],
])

function normalizeText(text) {
  let normalized = text.trim().toLowerCase();
  for (const [regex, replacement] of CHARACTER_REPLACEMENTS.entries()) {
     normalized = normalized.replace(regex, replacement)
  }

  return normalized;
}
...

return BLOCKLIST.has(normalizeText(link.text));
```

DISCLAIMER: untested